### PR TITLE
add updated logo

### PR DESCRIPTION
**Pull request by timglorioso**
Willing to make edits if desired!

### CODE REVIEW

**Comment by timglorioso** [comment]
I edited the readme on my fork so you can preview it 👍

**Comment by SD10** [comment]
@omaralbeik @nathantannar4 @MacMeDan What do you guys think? I like it but it could change again based on further README changes. 

Really appreciate you making these changes @timglorioso 💯 🎉

**Review by nathantannar4 — DISMISSED** [review]
I like this!

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <p>
-  <img src="https://cdn.rawgit.com/MessageKit/MessageKit/master/Assets/mklogo.svg" title="MessageKit logo">
+  <img src="https://raw.githubusercontent.com/timglorioso/MessageKit/master/Assets/messagekit-logo.png" title="MessageKit logo">
```
Comment (SD10) [inline README.md]: @timglorioso Can we change this back and I'll merge the PR?

**Review by SD10 — CHANGES_REQUESTED** [review]
See comments

File: README.md (hunk 1 above)
Comment (timglorioso) [reply README.md]: @SD10 Assuming you want the URL updated to reflect the new filename?

Comment (SD10) [reply README.md]: You could also just rename the file to match the old one 😃

**Comment by timglorioso** [comment]
I went ahead and kept using `raw.githubusercontent.com` because the RawGit CDN [permanently caches based on filename](https://github.com/rgrove/rawgit/blob/master/FAQ.md#how-long-does-the-cdn-cache-files-how-can-i-make-it-refresh-my-file), so we'd have to keep changing it (and the img src) for any future iterations 😉

**Comment by SD10** [comment]
@timglorioso Any reason we are using a `.png` over the previous `.svg`?

**Comment by timglorioso** [comment]
@SD10 The SVG format doesn't preserve text data as paths, so fonts don't render correctly if they can't be found client-side. I could force the text into a path if you'd like, shouldn't be hard 😁

**Review by SD10 — APPROVED** [review]
@timglorioso No need. Thanks for explaining and all your iterations 😃

**Pull request merged**